<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageExpansion</class>
 <widget class="QWidget" name="ImageExpansion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageExpansion</string>
  </property>
  <widget class="QSlider" name="SaturationBar">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>590</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>254</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="LightnessBar">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>620</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>254</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="HueBar">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>560</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="MakeButton">
   <property name="geometry">
    <rect>
     <x>919</x>
     <y>650</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>make expanded image</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="HueBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SaturationBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>590</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>254</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="LightnessBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>620</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Hue_lower</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturation_lower</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lightness_lower</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>561</x>
     <y>560</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Hue_upper</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="HueBox_2">
   <property name="geometry">
    <rect>
     <x>1041</x>
     <y>560</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>180</number>
   </property>
  </widget>
  <widget class="QSlider" name="HueBar_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="SaturationBox_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>590</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSlider" name="SaturationBar_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>590</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>590</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturation_upper</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="LightnessBox_2">
   <property name="geometry">
    <rect>
     <x>1041</x>
     <y>620</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>561</x>
     <y>620</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lightness_upper</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="LightnessBar_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>620</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="sliderPosition">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="RectImageButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>650</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>show rect image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Gamma</string>
   </property>
  </widget>
  <widget class="QSlider" name="GammaBar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>650</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="GammaBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>650</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
